common/scala/src/main/scala/whisk/core/containerpool/Container.scala

```diff
@@ -166,16 +170,20 @@ trait Container {
     implicit transid: TransactionId): Future[RunResult] = {
     val started = Instant.now()
     val http = httpConnection.getOrElse {
-      val conn = new HttpUtils(s"${addr.host}:${addr.port}", timeout, 1.MB)
+      val conn = if (config.poolingClient) {
```

Comment:
`poolingClient` seems an odd name since `HttpUtils` does pool as well. Should we rename to `newContainerClient`? (akka themselves did the same when they implemented the new ConnectionPool)